### PR TITLE
Dumping ceph branch into master.

### PR DESCRIPTION
**Pull request by eile**
This PR is to merge pending work before branching the code off into
keyv. Ceph code needs more testing, but was functional six months ago. I
expect this to be taken up again once the Lugano SSD pool is online.

Also includes C++11 usage in persistentMap

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -42,6 +42,8 @@ if(NOT COMMON_USE_CXX03)
 endif()
 common_find_package(Servus REQUIRED)
 common_find_package_post()
+#set(RADOS_FOUND TRUE) # TBD RADOS finder
```
Comment (tribal-tec) [inline CMakeLists.txt]: no pkgconfig for rados?

File: tests/persistentMap.cpp [hunk 2]
```diff
@@ -15,7 +15,7 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
-#define TEST_RUNTIME 240 //seconds
+#define TEST_RUNTIME 2400 //seconds
```
Comment (tribal-tec) [inline tests/persistentMap.cpp]: Against!

Comment (eile) [inline tests/persistentMap.cpp]: Ok, will revert. Don't remember why anyways

**Comment by eile** [comment]
Retest this please.

**Pull request merged**